modules/tenant-management/src/Volo.Abp.TenantManagement.Web/Pages/TenantManagement/Tenants/CreateModal.cshtml.cs

```diff
@@ -33,6 +33,17 @@ public class TenantInfoModel
             [StringLength(TenantConsts.MaxNameLength)]
             [Display(Name = "DisplayName:TenantName")]
             public string Name { get; set; }
+
+            [Required]
+            [StringLength(TenantConsts.MaxNameLength)]
+            [Display(Name = "DisplayName:AdminEmailAddress")]
```

Comment:
Display attribute should not be needed, because ABP automatically does this. Also, remove from others and test if the localization is still working.